#! /bin/sh /usr/share/dpatch/dpatch-run
## 14_fix_window_title.dpatch by Goedson Teixeira Paixao <goedson@debian.org>
## based on the patch contributed in the savannah gpaint bug tracker
## by Yavor Doganov (https://savannah.gnu.org/bugs/?18842)
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Do not show  the .po header in the window title.

@DPATCH@
diff -urNad gpaint-0.2.4+0.3.0pre5~/src/drawing.c gpaint-0.2.4+0.3.0pre5/src/drawing.c
--- gpaint-0.2.4+0.3.0pre5~/src/drawing.c	2005-01-27 00:48:45.000000000 -0200
+++ gpaint-0.2.4+0.3.0pre5/src/drawing.c	2007-04-20 19:13:29.000000000 -0300
@@ -301,7 +301,7 @@
     gchar *basename;
     
     /* Display the filename in the title bar. */
-    title = g_string_new(_(""));
+    title = g_string_new("");
     if (drawing->untitled)
     {
        g_string_append(title, _("Untitled"));     
